src/integration/common/header.js

```diff
@@ -8,12 +8,21 @@ export default () => {
       expect(logo.parentNode.textContent).toMatchSnapshot();
     });
 
+    it("I can see the offscreen text with product's language code set to English", () => {
+      const langCode = document.querySelector(
+        'header div span span[lang="en-GB"]',
+      );
+      expect(langCode).toBeInTheDocument();
+    });
+
     it('I can see the navigation', () => {
       const navigationItemEls = document.querySelector(
         'header nav ul > li > a',
       );
 
       expect(navigationItemEls).toBeInTheDocument();
+      expect(navigationItemEls.getAttribute('href')).toBeTruthy();
+      // expect(navigationItemEls.getAttribute('text')).toBeTruthy();
```

Comment:
@RichardPK are you able to look into this why this assertion wouldn't work?anything to do with the syntax?

Reply:
Updated 👍